### ISSUE TITLE
Select text by font, color, AND justification

### ISSUE THREAD

**Issue by Ridilla** [wonderbit/sketch-select-similar-layers#32]
Right now I use the plugin to mass select text to convert to text styles and it would be great if we could also match on text justification.

**Comment by phildziedzic** [comment]
Yes! Would love this as well to batch apply my design system updates to other symbols. Justification is the other key text attribute :)
